cla_backend/apps/legalaid/tests/test_views.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client
+from django.contrib.auth.models import Permission
+from django.core.urlresolvers import reverse
+
+from legalaid.views import FullCaseViewSet
+from cla_backend.apps.call_centre.permissions import *
+from cla_backend.urls import *
+from rest_framework import routers
+from legalaid.tests.views.test_base import CLAOperatorAuthBaseApiTestMixin
+
+
+class FullCaseViewSetTestCase(CLAOperatorAuthBaseApiTestMixin,TestCase):
+    def setUp(self):
+        super(FullCaseViewSetTestCase, self).setUp()
+        self.url = reverse('call_centre:case-list')
+
+    def test_filter_queryset_for_unicode_characters_status_code_200(self):
```

Comment:
Something like `test_search_unicode` and `test_search_ascii` is neater and better conveys the high-level intent.